packages/node/src/indexer/api.service.ts

```diff
@@ -158,16 +159,42 @@ export class CosmosClient extends CosmWasmClient {
   */
 
   async blockInfo(height?: number): Promise<Block> {
-    return this.getBlock(height);
+    try {
+      return await this.getBlock(height);
+    } catch (e) {
+      return retryRequest(
+        e,
+        `Failed to fetch blockInfo at ${height}`,
+        height,
+        this.blockInfo.bind(this),
+      );
+    }
```

Comment:
I think the try/catch can move into retryRequest. Then this function would become:

```suggestion
    return retryRequest(this.blockInfo.bind(this, blockHeight), `Failed to fetch blockInfo at ${height}`);
```